qemu/tests/cfg/device_assignment.cfg

```diff
@@ -1,13 +1,26 @@
 - device_assignment:
     only pf_assignable
     virt_test_type = qemu
+    pre_command = "modprobe vfio-pci"
```

Comment:
why it was not called ?   From my checking,  it will be called if host_set_flag was set to setup env.